inc/class-registration.php

```diff
@@ -961,6 +966,31 @@ public static function sticky_style() {
 		echo '<style id="o-sticky-inline-css">.o-sticky.o-sticky-float { height: 0px; } </style>';
 	}
 
+	/**
+	 * Add inline script for temporary save mode.
+	 *
+	 * @static
+	 * @access public
+	 */
+	public static function test_add_temp_save() {
+
+		/**
+		 * In this function we add a script that will activate the temporary save mode.
+		 * This will be for the Pro users. This will be removed in the future and moved to the Pro plugin.
+		 */
```

Comment:
Instead of doing this, can't we just do it based on if Stripe block exists in the form content area?

Reply:
We can. This is a placeholder for testing.